langchain/utilities/duckduckgo_search.py

```diff
@@ -0,0 +1,62 @@
+"""Util that calls DuckDuckGo Search.
+
+No setup required. Free.
+https://pypi.org/project/duckduckgo-search/
+"""
+from typing import Dict, List
+
+from duckduckgo_search import ddg
+from pydantic import BaseModel, Extra
+
+
+class DuckDuckGoSearchAPIWrapper(BaseModel):
+    """Wrapper for DuckDuckGo Search API.
+
+    Free and does not require any setup
+    """
+
+    k: int = 10
+
+    class Config:
+        """Configuration for this pydantic object."""
+
+        extra = Extra.forbid
+
+    def run(self, query: str) -> str:
+        """Run query through DuckDuckGo and return results."""
+        results = ddg(query, region="wt-wt", safesearch="Off", time="y")
```

Comment:
These could be arguments to the class. Also it seems reasonable to add the  `max_results` argument here. I'd deafult to safesearch being on for the general repo as well (can make it configurable)